.github/workflows/website.yml

```diff
@@ -1,5 +1,8 @@
 name: Update website
 
+concurrency:
+  group: mesonbuild.com
```

Comment:
I'm a bit worried what happens if it gets stopped during the upload of the updated website? I think the upload it a `git push`, is that atomic operation?

Reply:
It should *not* cancel anything since the flag is not to set to `true` here. The way it is currently set up is that only one website job can run at a time. All other jobs are paused/pending until the previous one is done.

See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency

Reply:
Oh, I see, you don't have `cancel-in-progress` here, so that's going to queue every website updates. That's actually super important fix, nice catch!

Reply:
I have also added a comment so it doesn't get added by accident.